tests/test_provision.py

```diff
@@ -188,5 +189,7 @@ def test_provision_no_recreate_json(tox_project: ToxProjectCreator) -> None:
     assert msg in result.out
     with (project.path / "out.json").open() as file_handler:
         requires = json.load(file_handler)
-    version = __version__.split("+")[0]
+    # __version__ could be something like 4.0.0b3.dev23+g50e62f93 so we still  # noqa: SC100
+    # need to sanitize it to keep the base and add "a0"  # noqa: SC100
+    version = Version(__version__).base_version + "a0"
```

Comment:
Use f-string rather than +

Reply:
Sure. Re spelling, if you are curious, I am using two other solutions in other projects, [sometimes both](https://github.com/ansible/ansible-lint/blob/main/.pre-commit-config.yaml#L45) as they can also share the same custom dictionary, which I store into `.config/dictionary.txt`, one is cspell and the other is spellcheck-py.

Reply:
I'm happy with the current implementation.